src/solvers/lowering/byte_operators.cpp

```diff
@@ -118,6 +119,26 @@ static exprt unpack_rec(
         array.copy_to_operands(op);
     }
   }
+  else if(src.id() == ID_string_constant)
```

Comment:
I might be wrong, but I think these are not used by Java, so the commit message may need amending

Reply:
Amended to not have that unnecessary restriction, but I do see results of `git grep string_constantt` in `jbmc/` as well...

Reply:
They are used for `input` and `output` steps only; the Java bytecode parser makes them to describe string literals, but those are later transformed into references to constant `java.lang.String` instances, so there are no free-floating constants in the same way as there are in C.

Reply:
Ah, ok! I guess it must have been some C code where I had seen them in the wild.